### PR TITLE
Added more information about Autoconfiguration

### PR DESCRIPTION
**Pull request by ryanleesipes**
Added more information about the autoconfiguration and setup process (around where info gets sent).

### CODE REVIEW

**Comment by ryanleesipes** [comment]
This change was asked for by our Engineering Manager, please advise if it makes sense.

File: thunderbird_privacy_policy/en-US.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ __Technical data for updates__: Thunderbird periodically connects to our server
...
 
-__DNS servers, Standard Autoconfiguration URIs, and Mozilla's Configuration Database__: To simplify the email set-up process, Thunderbird tries to determine the correct settings for your account by contacting Mozilla’s configuration database as well as external servers. These include DNS servers and standard autoconfiguration URIs. During this process, Thunderbird may send your email domain to these servers, but your full email address and password are never disclosed.
+__DNS servers, Standard Autoconfiguration URIs, and Mozilla's Configuration Database__: To simplify the email set-up process, Thunderbird tries to determine the correct settings for your account by contacting Mozilla’s configuration database as well as external servers. These include DNS servers and standard autoconfiguration URIs. During this process your domain will be sent to to Mozilla's configuration database. Your email may be disclosed to your network administrators. Your email address and password may be disclosed to administrators of your domain.
```
Comment (peiying2) [inline thunderbird_privacy_policy/en-US.md:66]: - Add a comma between process and you so it looks like this: `... URIs. During this process, your domain will...`
- Remove an extra "to" in `your domain will be sent to Mozilla's configuration database.`
- Add a "the" in the last sentence so it reads like this: `... be disclosed to the administrators of your domain.`

**Comment by peiying2** [comment]
Please have a legal person to review this. Add @litkaemily.

**Comment by litkaemily** [comment]
Closing this pull request and will open a new one. Making the changes as discussed over email.

**Pull request closed without merging**